### PR TITLE
Create environment variables for blob object expiration

### PR DESCRIPTION
**Pull request by calvinnhieu**
These changes introduce 2 new environment variables:
- `DSS_BLOB_TTL`: Number of days that "expirable" blobs are allowed to live in cloud storage (time to live)
- `DSS_BLOB_PUBLIC_TTL`: Number of days that we publicly claim that blobs live in cloud storage for

Some examples of "expirable" blobs are blobs in test buckets and/or checkout buckets. It is up for discussion as to whether we should differentiate TTLs for different uses (GS vs S3, different bucket types), however I don't have an argument for this. One potential case raised by @ttung is if we need to be cost conscious between different cloud providers.

### Deployment instructions & migrations
Ensure that `source environment` is run before deploying.

Related to #1498

### CODE REVIEW

**Comment by kislyuk** [comment]
It's probably better to put the unit into the var name: `DSS_BLOB_PUBLIC_TTL_DAYS`

File: .python-version [hunk 1]
```diff
@@ -0,0 +1,2 @@
+data-store
```
Comment (ttung) [inline .python-version]: I don't think you should be checking this in.  You should put this in your private gitignore (i.e., `.git/info/exclude`).

Comment (calvinnhieu) [reply .python-version]: Agreed. My mistake!

Comment (calvinnhieu) [reply .python-version]: My bad. Thanks for the tip re: private gitignore!

**Pull request merged**